go.mod

```diff
@@ -1,6 +1,6 @@
 module k8s.io/release
 
-go 1.23
+go 1.22.4
```

Comment:
I assume that's the only change we need. Let's keep go 1.23 in GitHub actions to not revert the whole patch.

Reply:
agree

Reply:
also maybe we can but to the latest go 1.22 as well

Reply:
no, it doesn't work: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/release/3781/pull-release-image-k8s-ci-builder/1844259485523120128